metastats-infrastructure/src/test/groovy/life.qbic.metastats/OpenBisSearchSpecification.groovy

```diff
@@ -1,58 +1,80 @@
 package life.qbic.metastats
 
-import life.qbic.metastats.datamodel.MetaStatsSample
-import life.qbic.metastats.request.OpenBisSearch
-import life.qbic.metastats.request.OpenBisSession
+
 import spock.lang.Specification
 
-class OpenBisSearchSpecification extends Specification{
+class OpenBisSearchSpecification extends Specification {
```

Comment:
Is there a reason for commenting out this test?

Reply:
see other comment